liquibase-integration-tests/src/test/resources/liquibase/liquibase.integrationtest.properties

```diff
@@ -62,7 +62,7 @@ integration.test.informix.url=jdbc:informix-sqli://localhost:9090/liquibase:info
   DB_LOCALE=en_us.utf8;GL_DATE=%iY-%m-%d
 # Microsoft SQL Server
 # @todo There are two special integration tests for MSSQL Server: MssqlCaseSensitive and MssqlJtds. We need an extension to this naming mechanism to allow these tests to use different URLs.
-integration.test.mssql.url=jdbc:sqlserver://localhost:1433;databaseName=lbcat
+integration.test.mssql.url=jdbc:sqlserver://localhost:14333;databaseName=lbcat
```

Comment:
@PavloTytarchuk for consistency sake shouldn't this actually reference the container name and port like we do for other database platforms?

Reply:
Hello @jnewton03, this is **liquibase.integrationtest.properties** file and I changed port to 14333 to match port from docker-compose file:  - "14333:1433". 
In **liquibase.integrationtest.jenkins.properties** file it has reference to containername and port:
**integration.test.mssql.url=jdbc:sqlserver://mssql:1433;databaseName=lbcat**

Reply:
According to logs, tests are running on Jenkins.
https://jenkins.datical.net/job/liquibase-pro/job/DAT-4893/21/consoleFull

06-16 15:21:43  [ThreadedStreamConsumer] INFO org.apache.maven.plugin.surefire.SurefirePlugin - Running liquibase.dbtest.mssql.MssqlIntegrationTest
06-16 15:26:24  [ThreadedStreamConsumer] INFO org.apache.maven.plugin.surefire.SurefirePlugin - Tests run: 32, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 281.016 s - in liquibase.dbtest.mssql.MssqlIntegrationTest

Reply:
🤕 my bad.  i misread this completely, @PavloTytarchuk !